test/e2e-tests.sh

```diff
@@ -46,12 +46,12 @@ TEST_YAML_IGNORES=${TEST_YAML_IGNORES:-""}
 
 # Allow ignoring some yaml tests, space separated, should be the basename of the
 # test for example "s2i"
-TEST_TASKRUN_IGNORES=${TEST_TASKRUN_IGNORES:-""}
+TEST_TASKRUN_IGNORES=${TEST_TASKRUN_IGNORES:-"cloudevent"}
```

Comment:
Should we had a comment about this one ? (aka depend on `ko`, should use a released image in there — we should also create an issue about it)

Reply:
I am creating an issue for this and also let @iancoffey know :D

I think he is already working, I see a PR #396

Reply:
Created issue https://github.com/tektoncd/catalog/issues/422